lib/dns/src/opstrace.ts

```diff
@@ -84,11 +84,10 @@ async function loginPollRequest(
   let data: any;
   try {
     data = JSON.parse(resp.body);
-  } catch (err: any) {
+  } catch (err) {
     log.warning(
-      "could not parse response body as JSON: %s: %s",
-      err.code,
```

Comment:
JSON.parse throws `SyntaxError`, which does not have a `code` member. See [mdn](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse#exceptions).